### DISCUSSION TITLE
@types/bezier-js [bezier-js] .get() method returns a different structure than Point

### DISCUSSION THREAD

**Discussion by brandonscript** [DefinitelyTyped/DefinitelyTyped#63993] [Issues with a @types package]
Per https://github.com/Pomax/bezierjs/blob/d19695f3cc3ce383cf38ce4643f467deca7edb92/src/utils.js#L148 bezier-js doesn't return a `Point` interface here, it returns a `{ x, y, t }` object instead — which is a type that does not currently exist.

Not sure how much of the project this affects, but figured I'd start with an issue and go from there. I thought about just updating Point to return a `t` prop, but Point is used as the input into several functions as well, like the main Bezier() init.
<img width="792" alt="Screenshot 2023-01-19 at 10 32 17 AM" src="https://user-images.githubusercontent.com/1480253/213543178-f47a42dd-4357-4f2a-ab64-d0d072a7ce93.png">
<img width="621" alt="Screenshot 2023-01-19 at 10 32 05 AM" src="https://user-images.githubusercontent.com/1480253/213543185-497ff953-4fd2-402c-b0a2-afa6671b504a.png">
<img width="485" alt="Screenshot 2023-01-19 at 10 31 37 AM" src="https://user-images.githubusercontent.com/1480253/213543188-a1580237-6813-4c0b-b70a-f8c9d3214e15.png">

[a comment by an automated account is not shown]

**Comment by danmarshall** [comment]
It's unclear. The types are written for [version 4](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bezier-js/index.d.ts#L1) but the library is at [version 6](https://github.com/Pomax/bezierjs/blob/d19695f3cc3ce383cf38ce4643f467deca7edb92/package.json#L3). Might require an investigation.
